.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: Feature Request
+description: Suggest a new feature or enhancement for the REMIND model.
+title: '[FEATURE REQUEST]'
```

Comment:
Just no.  That is what the `labels` are for.

Reply:
Same as [above](https://github.com/remindmodel/remind/pull/1521#discussion_r1453446782)